### ISSUE TITLE
[BUG]schema became case insensitive when create property on it.

### ISSUE THREAD

**Issue by lightjiao** [orientechnologies/orientdb#7955]
### OS: centos 6.8

execute sql below
```
create property v._version integer
```
```
update v content {
            "name": "a",
            "_version": 1,
            "_VERSION": 3
        } 
where @rid = 10:1
```

and then select the record
```
select from v where @rid = 10:1
```

## Expected behavior  
Expect got both `_version` and `_VERSION` schema:
```
        {
            "@type": "d",
            "@rid": "#10:1",
            "@version": 5,
            "@class": "V",
            "out_": [],
            "name": "a",
            "_version": 1,
            "_VERSION": 3,
            "@fieldTypes": "out_=g"
        }
```

## Actual behavior  
Actual the `_VERSION` schema were gone:
```
        {
            "@type": "d",
            "@rid": "#10:1",
            "@version": 6,
            "@class": "V",
            "out_": [],
            "name": "a",
            "_version": 1,
            "@fieldTypes": "out_=g"
        }
```

## Steps to reproduce  
If don't create the property, every thing was fine. but create a property, the schema became case insensitive.
it's a bug? or a feature?

**Comment by luigidellaquila** [comment]
Hi @lightjiao 

It's a known issue in v 2.2, it's hard to change this behavior without breaking a lot of things.
Good news is that we already addressed it in v 3.0.

Thanks

Luigi

**Comment by lightjiao** [comment]
ok....fine.....
